### ISSUE TITLE
Does this run under Raspberry pi Linux?

### ISSUE THREAD

**Issue by kevinl8890** [nanovna/NanoVNA-QT#2]

**Comment by nanovna** [comment]
I don't see why it shouldn't; try compiling it from source.

**Comment by dl9sec** [comment]
Yes. See https://www.experimental-engineering.co.uk/2020/07/04/nanovna-qt-raspberry-pi-appimage-build/.
Works very well, but not from the latest development sources...
